LabExtended/Patches/Fixes/Scp079RecontainIgnoreNpcPatch.cs

```diff
@@ -10,15 +10,18 @@ namespace LabExtended.Patches.Fixes
     [HarmonyPatch(typeof(Scp079Recontainer), nameof(Scp079Recontainer.OnServerRoleChanged))]
     public static class Scp079RecontainIgnoreNpcPatch
     {
-        public static bool Prefix(Scp079Recontainer __instance, ReferenceHub hub, RoleTypeId newRole, RoleChangeReason reason)
-        {
+        public static bool Prefix(Scp079Recontainer __instance, ReferenceHub hub, RoleTypeId newRole)
+            {
+            if (!ExPlayer.TryGet(hub, out var player) || player.IsNpc)
+                return false;
+
             if (newRole != RoleTypeId.Spectator || !__instance.IsScpButNot079(hub.roleManager.CurrentRole))
                 return false;
 
-            if (!Scp079Role.ActiveInstances.Any(x => x.TryGetOwner(out var owner) && ExPlayer.TryGet(owner, out var player) && player.Switches.CanBeRecontainedAs079))
+            if (!Scp079Role.ActiveInstances.Any(x => x.TryGetOwner(out var owner) && ExPlayer.TryGet(owner, out var player) && !player.IsNpc && player.Switches.CanBeRecontainedAs079))
```

Comment:
Calling IsNpc is redundant as CanBeRecontainedAs079 SHOULD be set to false in NpcHandler.Spawn